langchain/vectorstores/vecdb.py

```diff
@@ -0,0 +1,43 @@
+"""Wrapper around ChromaDB embeddings platform."""
+import os
+import vecdb
+
+from typing import Iterable, Optional, List, Any, TypeVar, Type
+
+from langchain.vectorstores import VectorStore
+from langchain.embeddings.base import Embeddings
+from langchain.docstore.document import Document
+
+VST = TypeVar("VST", bound="VectorStore")
+
+
+class VecDB(VectorStore):
+    def __init__(self, dataset_id: str):
+        self.client = vecdb.Client(os.getenv("VECDB_API_KEY"))
+        self.dataset = self.client.create_dataset(dataset_id)
+
+    def add_texts(
+        self,
+        texts: Iterable[str],
+        metadatas: Optional[List[dict]] = None,
+        **kwargs: Any,
+    ) -> List[str]:
+        self.dataset.insert(data=texts, metadata=metadatas, **kwargs)
```

Comment:
this method is expected to return a list of string ids. what's the output of dataset.insert?